packages/headless/src/controllers/core/search-box/headless-core-search-box.ts

```diff
@@ -137,6 +137,11 @@ export interface SearchBoxState {
    * Determines if a query suggest request is in progress.
    */
   isLoadingSuggestions: boolean;
+
+  /**
+   * The query suggestions id.
+   */
+  querySuggestionId: string;
```

Comment:
In this context, this is not actually the query suggestion ID, but rather the search box ID, so I'd rename the property to `id` and document it as follows:

```
The search box ID.
```

I think it's OK to expose this property in the controller state, since the search box ID will be randomly generated unless explicitly passed as a prop when building the controller, and this information is necessary in order to manually dispatch query suggestion actions and query set actions on the correct search box controller.

Another (complementary) possibility would be to add a `clearSuggestions` method directly on the controller.

Reply:
I'm OK with not adding a method.

I disagree with the naming, however. As I said, this property could also be used when dispatching query set actions (which don't relate to query suggestions). In the context of the search box controller, it really corresponds to the search box ID.

One thing we could do would be to update the `RegisterQuerySuggestActionCreatorPayload`, `ClearQuerySuggestActionCreatorPayload`, and `FetchQuerySuggestionsActionCreatorPayload` documentation in querysuggest-actions.ts to indicate that the `id` property should be the same as the ID of the "parent" search box controller.

I.e.,

```
[A|The] unique identifier for the [target|new] query suggest entity (e.g., `search_box1dq32`). Usually, this will be the ID of the search box controller that requests the query suggestions.
```